dataland-frontend/tests/e2e/specs/high/Prepopulation.ts

```diff
@@ -1,84 +1,98 @@
-describe('Population Test',  () => {
-    Cypress.config({
-        defaultCommandTimeout: 480000
-    })
+import { doThingsInChunks } from "../../support/utility";
 
-    let eutaxonomiesData: any
-    let companiesData: any
-    const companyAssociatedEuTaxonomyData: any = []
+const chunkSize = 40
+describe('Population Test',
+    () => {
+        Cypress.config({
+            defaultCommandTimeout: 900 * 1000
+        })
 
-    before(function () {
-        cy.fixture('EuTaxonomyData').then(function (eutaxonomies) {
-            eutaxonomiesData = eutaxonomies
-        });
-        cy.fixture('CompanyInformation').then(function (companies) {
-            companiesData = companies
+        let eutaxonomiesData: any
+        let companiesData: any
+        const companyAssociatedEuTaxonomyData: any = []
+
+        before(function () {
+            cy.fixture('EuTaxonomyData').then(function (eutaxonomies) {
+                eutaxonomiesData = eutaxonomies
+            });
+            cy.fixture('CompanyInformation').then(function (companies) {
+                companiesData = companies
+            });
         });
-    });
 
-    async function uploadData(dataArray: Array<object>, endpoint: string) {
-        console.time(`The elapsed time to upload ${dataArray.length} items for '${endpoint}'`)
-        const chunkSize = 80;
-        for (let i = 0; i < dataArray.length; i += chunkSize) {
-            const chunk = dataArray.slice(i, i + chunkSize);
-            await Promise.all(chunk.map(async (element: object) => {
-                    await fetch(`${Cypress.env("API")}/${endpoint}`, {
-                        method: 'POST',
-                        headers: {
-                            'Content-Type': 'application/json'
-                        },
-                        body: JSON.stringify(element)
-                    }).then(response => {
-                        assert(response.status.toString() === "200",
-                            `Got status code ${response.status.toString()} for index ${i}. Expected: 200`)
-                    })
-                })
-            )
+        function uploadSingleElementOnce(endpoint: string, element: object): Promise<void> {
```

Comment:
diskutierbare Frage: für die Übersichtlichkeit wäre es vllt ganz gut functionen in die datei utility.ts auszulagern und in den test files nur tests zu lassen?